platform/pc/boot/stage1.s

```diff
@@ -75,11 +75,42 @@ dap:
         .lba          dq 1
 
 
+%ifdef DEBUG
+bits 16
+%include "debug.inc"
+bits 32
```

Comment:
We don't want to go to bits 32 here...just the include without either bits statement is fine.